### DISCUSSION TITLE
Who fell in 💘 with their own reflection in 🇬🇷 mythology?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#1138] [Galaxy Trivia] [answered]
**Bonus:** What mythological creature is Mr. Tumnus in Narnia?

**Comment by peterbaccaro** [comment]
In Greek mythology, **Narcissus** dies by a pool gazing at his own reflection that he falls in love with. He has no concern about anything around him nor does he eat or sleep.

Tumnus is a **faun**. Fauns are creatures from Roman mythology that are half goat and half human, the bottom of them being goat and the top being human.

**Comment by thuongtruong1009** [comment, marked as answer]
n Greek mythology, the individual who fell in love with their own reflection is Narcissus.

Mr. Tumnus, in the world of Narnia created by C.S. Lewis, is a faun. Fauns are mythological creatures that are often depicted as having the upper body of a human and the lower body of a goat, along with horns on their heads.
